Federal Data/Nepal Rastra Bank/Registered saving account of 77 district.csv

```diff
@@ -0,0 +1,79 @@
+District,Total number of registered saving accounts,Total population,Registered saving accounts penetration (%)

```

Comment:
Can we have a gloss somewhere, perhaps in a README, that describes the meaning of terms like what the "penetration" of saving accounts means, and what counts as active, etc.? That will make it easier to consume these data. Could we also have link to the original data set to make verification and review easy?